libs/core-ui/src/lib/Cohort/Cohort.ts

```diff
@@ -63,9 +63,10 @@ export class Cohort {
 
   public sort(
     columnName: string = JointDataset.IndexLabel,
-    reverse?: boolean
+    reverse?: boolean,
+    force?: boolean
   ): void {
-    if (this.currentSortKey !== columnName) {
+    if (this.currentSortKey !== columnName || force) {
```

Comment:
So just to clarify: In the `force` (only) case the column name would be the same as the current sort key, but it's not sorted yet? Is there ever a reason not to sort? If not, we should probably just sort whenever the current sort key changes (?)

Reply:
"In the force (only) case the column name would be the same as the current sort key, but it's not sorted yet"
exactly, I think it was in reverse order
"we should probably just sort whenever the current sort key changes"
yes, in this case sort key remained the same, but I think it was in reverse order